README.md

```diff
@@ -393,6 +401,6 @@ Before contributing to this project, please read the
 
 ## License
 
-Copyright © 2018-2020 Jean-Philippe Cugnet and Contributors
+Copyright (c) 2018-2021 Jean-Philippe Cugnet and Contributors
```

Comment:
Please bring the `©` back, here we are in Markdown, with a properly formatted text, so let’s use the full power of UTF-8.